config/config.go

```diff
@@ -182,6 +182,10 @@ type Config struct {
 	// 1. there is a network partition problem between TiDB and PD leader.
 	// 2. there is a network partition problem between TiDB and TiKV leader.
 	EnableForwarding bool `toml:"enable-forwarding" json:"enable-forwarding"`
+	// EnableLocalTxn determines whether the @@txn_scope will be set to "local" by default if labels.zone is configured.
+	// This is used to decouple Stale Read and Local Txn, make sure they can be used independently.
+	// NOTICE: This filed may be removed when Local Txn is released.
+	EnableLocalTxn bool `toml:"enable-local-txn" json:"enable-local-txn"`
```

Comment:
I think this should be a `global varaible` instead of instance configuration.

Reply:
Yep, +1. Replacing the variable `TiDBTxnScope` into `EnableLocalTxn`.

Reply:
This could be done later.

Reply:
I have changed it into a system variable. PTAL, thx.